packages/newspringchurchapp/src/prayer/PrayerPreviewCard/index.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+import PropTypes from 'prop-types';
+import Color from 'color';
+import {
+  Avatar,
+  BodyText,
+  Button,
+  Card,
+  CardContent,
+  H3,
+  H6,
+  PaddedView,
+  styled,
+} from '@apollosproject/ui-kit';
+
+// TODO: Borrowed `Overlay` and `getGradientValues` from <GradientOverlayImage />
+// We should probably extract these to someplace else and import them in both places.
+const Overlay = styled(StyleSheet.absoluteFillObject)(LinearGradient);
+
+const getGradientValues = (overlayColor) => {
+  let colors;
+  if (overlayColor.length > 1) {
+    colors = [
+      `${Color(overlayColor[0])
+        .fade(0.6)
+        .string()}`,
+      `${Color(overlayColor[1])
+        .fade(0)
+        .string()}`,
+    ];
+  } else {
+    colors = [
+      `${Color(overlayColor[0])
+        .fade(1)
+        .string()}`,
+      overlayColor[0],
+    ];
+  }
+  const values = {
+    colors,
+    start: { x: 0, y: 0 },
+    end: { x: 0, y: 1 },
+    locations: [0.25, 0.75],
+  };
+
+  return values;
+};
+
+const StyledCard = styled(() => ({
+  marginHorizontal: 0,
+  marginVertical: 0,
+}))(Card);
+
+const StyledCardContent = styled(() => ({
+  alignItems: 'center',
+}))(CardContent);
+
+const StyledBodyText = styled(({ theme }) => ({
+  marginTop: theme.sizing.baseUnit * 1.5,
```

Comment:
probably should add `margin` to a container around these according to our other convos instead of the text itself.

Reply:
This is a good call. Moved the margin to a container.